apps/report-documents-not-received/index.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
     '/report-submitted': {
       backLink: false,
       clearSession: true
-    }
+    },
```

Comment:
Any specific reason we decided to put this inside the DNR flow instead of the rod main flow?

Reply:
No, I was thinking it would need to go in all 3 flows but never ended up being the case. Do you think I should move it?

Reply:
Yes, I think it would make sense to move this inside rod main flow (though that would not make any functional difference).